change emphasis of text (#148)
bcgov/aqwarnings: frontend/index.qmd

- Subscribe to the [Air Quality Subscription Service](https://www2.gov.bc.ca/gov/content/environment/air-land-water/air/air-quality/air-advisories/air-quality-subscription-service)
- Outside regular business hours, visit [Environment and Climate Change Canada’s Public Weather Alerts for B.C.](https://weather.gc.ca/?province=bc&zoom=5&alertTableFilterProv=BC)

> - For information about air quality warnings within **Metro Vancouver** and parts of the **Fraser Valley Regional District** ([see map](https://gis.metrovancouver.org/maps/Air)), visit [Metro Vancouver’s Air Quality Data and Warnings](https://metrovancouver.org/services/air-quality-climate-action/air-quality-data-and-advisories) webpage.

Reviewer: ```suggestion
- Visit [Metro Vancouver’s Air Quality Data and Warnings](https://metrovancouver.org/services/air-quality-climate-action/air-quality-data-and-advisories) for information about **Metro Vancouver** and parts of the **Fraser Valley Regional District** ([see regions in AirMap](https://gis.metrovancouver.org/maps/Air))
```

Or maybe this? Is the intent that they go to the warnings page or the AirMap?
